java/ql/src/Security/CWE/CWE-200/TempDirUtils.qll

```diff
@@ -35,32 +35,6 @@ predicate isFileConstructorArgument(Expr expSource, Expr exprDest, int paramCoun
   )
 }
 
-/**
- * A `File` method where the temporary directory is still part of the root path.
- */
-private class TaintFollowingFileMethod extends Method {
-  TaintFollowingFileMethod() {
-    this.getDeclaringType() instanceof TypeFile and
-    this.hasName(["getAbsoluteFile", "getCanonicalFile"])
-  }
-}
-
-private predicate isTaintPropagatingFileTransformation(Expr expSource, Expr exprDest) {
-  exists(MethodAccess fileMethodAccess |
-    fileMethodAccess.getMethod() instanceof TaintFollowingFileMethod and
-    fileMethodAccess.getQualifier() = expSource and
-    fileMethodAccess = exprDest
-  )
-}
-
-/**
- * Holds if taint should propagate from `node1` to `node2` across some file creation or transformation operation.
- * For example, `taintedFile.getCanonicalFile()` is itself tainted.
- */
-predicate isAdditionalFileTaintStep(DataFlow::Node node1, DataFlow::Node node2) {
```

Comment:
Keep this as an empty deprecated predicate for now:
```
deprecated predicate isAdditionalFileTaintStep(DataFlow::Node node1, DataFlow::Node node2) { none() }
```

Reply:
OOI: Why would this be a breaking change?
Isn't this file unimportable by external users because it contains hyphens in it's path?